README.md

```diff
@@ -0,0 +1,5 @@
+# WWW ZAP Project
+
+The new home for the ZAP project which can be found on <https://www2.owasp.org/www-project-zap/>.
```

Comment:
`The OWASP home...`? Would avoid the need to remove the "new" later also emphasise OWASP as we plan to have another home/site.